contctme.html

```diff
@@ -0,0 +1,16 @@
+
+<body bgcolor="violet">
+<center>
+        <h1> Contact me </h1>
+        <li>
+            <b>
+                developer.shubham.rasal@gmail.com
```

Comment:
hello shubham,
Dont like this code..
too much white spaces

Reply:
okay i will fix it

Reply:
I will surely fix it into the next PR